sync/src/tests/synchronizer.rs

```diff
@@ -110,50 +113,46 @@ fn setup_node(thread_name: &str, height: u64) -> (TestNode, Shared) {
         let number = block.header().number() + 1;
         let timestamp = block.header().timestamp() + 1;
 
-        let last_epoch = shared.get_block_epoch(&block.header().hash()).unwrap();
+        let last_epoch = shared
+            .store()
+            .get_block_epoch(&block.header().hash())
```

Comment:
`shared.get_block_epoch`

Reply:
don't use ChainProvider::get_block_epoch, it will be deleted later